hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,10 @@
+RELEASE_TYPE: minor
+
+This release adds new deprecation warnings in
```

Comment:
The release type (minor) is perfect, but the note needs to describe all of the new deprecations.  You've made a great start, which I would extend to:

```
This release improves validation of the ``alphabet`` argument to the
:func:`~hypothesis.strategies.text` strategy.  The following misuses
are now deprecated, and will be an error in a future version:

- passing an unordered collection (such as ``set('abc')``), which
  violates invariants about shrinking and reproducibility
- passing an alphabet sequence with elements that are not strings
- passing an alphabet sequence with elements that are not of length one,
  which violates any size constraints that may apply

Thanks to Sushobhit for adding these warnings (:issue:`1329`).
```

Reply:
I intended to update release notes later, once all test passes, will now add it.
Will try to fix all points soon.